test/blackbox-tests/test-cases/melange/aliases.t

```diff
@@ -67,3 +90,17 @@ If user defines an alias, the default alias is not used for that stanza
   Error: Alias "melange" specified on the command line is empty.
   It is not defined in . or any of its descendants.
   [1]
+
+If user defines an alias, dune default alias no longer works
```

Comment:
I think we want the opposite behavior here.

Reply:
fixed in 72202babe